File: SpringBootJspAndRest/SpringBootJsp/src/main/java/pl/dmcs/rkotas/model/Student.java
package pl.dmcs.rkotas.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Student {
    @Id
    @GeneratedValue
    private long id;
    private String firstname;
    private String lastname;
    private String email;
    private String grades;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getGrades() {
        return grades;
    }

    public void setGrades(String grades) {
        this.grades = grades;
    }
}

File: student-list/src/app/student.ts
export class Stud {
  id: number;
  firstname: string;
  lastname: string;
  email: string;
  grades: string;
  course: string;
  year: string;


  constructor(firstname: string, lastname: string, email: string, grades: string, course: string, year: string) {
    this.firstname = firstname;
    this.lastname = lastname;
    this.email = email;
    this.grades = grades;
    this.course = course;
    this.year = year;
  }

}
